build/swivel/Swivel.sol

```diff
@@ -440,18 +440,22 @@ contract Swivel is ISwivel {
   }
 
   /// @notice Allows a user to cancel an order, preventing it from being filled in the future
-  /// @param o Order being cancelled
-  /// @param c Components of a valid ECDSA signature
-  function cancel(Hash.Order calldata o, Sig.Components calldata c) external returns (bool) {
-    bytes32 hash = validOrderHash(o, c);
+  /// @param o Array of offline orders being cancelled
+  function cancel(Hash.Order[] calldata o) external returns (bool) {
```

Comment:
as stated above, take an array
note we can remove the components if not checking the sig